integration/hurl/tests_ok/http_version_option.hurl

```diff
@@ -13,34 +22,94 @@ GET http://localhost:8000/http_version/10
 http1.0: true
 HTTP/1.1 200
 
+
+GET http://localhost:8000/http_version/10
+[Options]
+http1.0: true
+HTTP/1.1 200
+[Asserts]
+version == "1.1"
+
 # Default HTTP version for Flask is HTTP/1.1
 GET http://localhost:8000/http_version/11
 HTTP 200
 
+
+GET http://localhost:8000/http_version/11
+HTTP 200
+[Asserts]
+version == "1.1"
+
+
 GET http://localhost:8000/http_version/11
 [Options]
 http1.1: true
 HTTP 200
 
+
+GET http://localhost:8000/http_version/11
+[Options]
+http1.1: true
+HTTP 200
+[Asserts]
+version == "1.1"
+
+
 # The last option win
 GET http://localhost:8000/http_version/11
 [Options]
 http1.0: true
 http1.1: true
 HTTP 200
 
+
+GET http://localhost:8000/http_version/11
```

Comment:
We can remove this request, in the previous request we don't have an implicit version check (just `HTTP 200`) so no need to add an explicit check here.